tensorrt_llm/_torch/pyexecutor/py_executor_creator.py

```
@@ -344,6 +349,35 @@ def allocation_scope(current_stage: ExecutorMemoryType,
 
     validate_feature_combination(llm_args, model_engine, llm_args.sampler_type)
 
+    if llm_args.sm_disagg_config is not None:
+        if llm_args.cache_transceiver_config is not None:
+            raise ValueError(
+                "SM-level disaggregation is not compatible with disaggregated serving."
+            )
```

Ruff (0.14.3)

Avoid specifying long messages outside the exception class

(TRY003)